src/pages/task/taskTabs/Logs.tsx

```diff
@@ -66,22 +74,26 @@ interface GetLinksResult {
   rawLink?: string;
 }
 
-const getLinks = (logLinks: LogLinks, logType: LogTypes): GetLinksResult => {
+const getLinks = (
+  logLinks: LogLinks,
+  logType: LogTypes,
+  taskId: string,
+  execution: number
+): GetLinksResult => {
   if (!logLinks) {
     return {};
   }
```

Comment:
the buttons still show even if there no logs is that intended?
![image](https://user-images.githubusercontent.com/4605522/108377780-5ea6e980-71d2-11eb-90b3-cac89121d794.png)

Reply:
I now disable them if no logs are found

https://user-images.githubusercontent.com/10734386/109514852-743ecd80-7a74-11eb-940d-525649ce450b.mov